modules/SantoriniBoard.class.php

```diff
@@ -17,17 +17,21 @@ public function __construct($game)
    *       * 0 : don't add any field arg
    *       * 1 : arg is filled with singleton [z] (useful eg in argPlayerBuild)
    *       * 2 : arg is set to z (useful when building)
+   *   - optional $id : true to include the piece ID in the result array
    */
-  public static function getCoords($mixed, $arg = 0)
+  public static function getCoords($mixed, $arg = 0, $keepId = false)
```

Comment:
Since board.js now uses pieceId instead of x,y,z we need to keep the ID in some cases (e.g., argUsePower)

Reply:
Why not keeping the id everywhere then ?

Reply:
Yes, it is possible. We used this function a lot in the 'log' table to record the 'to' and 'from' when moving a piece. In that case, we have already piece ID as separate column. But no harm, I suppose :)